<launch>

  <node pkg="data_collector" type="GotoShelter.py" name="GotoShelter" output="screen" />

</launch>